esmvaltool/diag_scripts/ensclus/ensclus.py

```diff
@@ -0,0 +1,101 @@
+"""Ensemble Clustering Diagnostics.
+
+Author: Irene Mavilia (ISAC-CNR, Italy)
+Copernicus C3S 34a lot 2 (MAGIC)
+
+Description
+    Cluster analysis tool based on the k-means algorithm
+    for ensembles of climate model simulations
+Modification history
+    20181202-hard_jo: cleanup, style and finalising
+    20181002-arno_en: updating to version2_develpment (recipe/dataset)
+    20170710-mavi_ir: Routines written.
+"""
+
+# Basic Python packages
+import os
+import sys
+import logging
+import warnings
+import yaml
+import matplotlib.pyplot as plt
+from esmvaltool.diag_scripts.shared import\
+     (run_diagnostic, save_iris_cube, variables_available, extract_variables)
+
+plt.switch_backend('Agg')
```

Comment:
esmvaltool already sets the backend to Agg since PR #732, so there is no need to do it again here.

Reply:
Done in https://github.com/ESMValGroup/ESMValTool/pull/736/commits/98598793e4cb54f3f40e8b3f47e28ba1bab84daa